packages/syft/src/syft/core/tensor/autodp/phi_tensor.py

```diff
@@ -1318,8 +1359,12 @@ def min(self, axis: Optional[Union[int, Tuple[int, ...]]] = None) -> PhiTensor:
         return PhiTensor(
             child=result,
             data_subjects=self.data_subjects[indices],
-            min_vals=lazyrepeatarray(data=result.min(), shape=result.shape),
-            max_vals=lazyrepeatarray(data=result.max(), shape=result.shape),
+            min_vals=lazyrepeatarray(
+                data=self.min_vals.data, shape=result.shape, data_type="min_val"
```

Comment:
NITPICK:  I think we should include .min() on the data from previous code , same comment for the below max.